### PR TITLE
sushi rectification

### PR DESCRIPTION
**Pull request by jackhaibo**
**What this PR does / why we need it**:
Modify the error return code according to the latest `kubernetes-csi` document.

### CODE REVIEW

File: csi/server/plugin/opensds/controller.go [hunk 1]
```diff
@@ -534,17 +511,17 @@ func (p *Plugin) ControllerPublishVolume(ctx context.Context,
...
+	err = c.isVolumeCanBePublished(canAtMultiNode, attachReq, volSpec.MultiAttach)
 	if err != nil {
 		return nil, err
 	}
 
-	newAttachment, errAttach := client.CreateVolumeAttachment(attachReq)
+	newAttachment, errAttach := c.Cli.CreateVolumeAttachment(attachReq)
 	if errAttach != nil {
-		msg := fmt.Sprintf("the volume %s failed to publish to node %s, error info: %v",
+		msg := fmt.Sprintf("the volume %s is failed to publish to node %s, error info: %v",
```
Comment (leonwanghui) [inline csi/server/plugin/opensds/controller.go]: s/is failed to publish/is failed to be published

File: csi/server/plugin/opensds/controller.go [hunk 2]
```diff
@@ -568,21 +545,21 @@ func (p *Plugin) ControllerPublishVolume(ctx context.Context,
...
+		err = c.isVolumeCanBePublished(canAtMultiNode, attachReq, secondaryVolume.MultiAttach)
 		if err != nil {
 			return nil, err
 		}
 
-		newAttachment, errAttach := client.CreateVolumeAttachment(attachReq)
+		newAttachment, errAttach := c.Cli.CreateVolumeAttachment(attachReq)
 		if errAttach != nil {
-			msg := fmt.Sprintf("failed to ControllerPublishVolume: the volume %s failed to publish to node %s, error info %v",
+			msg := fmt.Sprintf("the volume %s failed to publish to node %s, error info %v",
```
Comment (leonwanghui) [inline csi/server/plugin/opensds/controller.go]: Same here

File: csi/server/plugin/opensds/controller.go [hunk 3]
```diff
@@ -594,44 +571,38 @@ func (p *Plugin) ControllerPublishVolume(ctx context.Context,
...
 		glog.Error(msg)
 		return nil, status.Error(codes.InvalidArgument, msg)
 	}
 
 	//check volume is exist
-	volSpec, errVol := client.GetVolume(req.VolumeId)
+	volSpec, errVol := c.Cli.GetVolume(req.VolumeId)
 	if errVol != nil || volSpec == nil {
-		msg := fmt.Sprintf("the volume %s is not exist, err: %v", req.VolumeId, errVol)
+		msg := fmt.Sprintf("the volume %s is not exist: %v", req.VolumeId, errVol)
```
Comment (leonwanghui) [inline csi/server/plugin/opensds/controller.go]: does not exist

File: csi/server/plugin/opensds/controller.go [hunk 4]
```diff
@@ -357,7 +341,7 @@ func (p *Plugin) DeleteVolume(
 
 // isStringMapEqual implementation
 func isStringMapEqual(metadataA, metadataB map[string]string) bool {
-	glog.V(5).Infof("start to isStringMapEqual, metadataA = %v, metadataB = %v!",
+	glog.V(5).Infof("start to determine whether the two map structures are equal, metadataA = %v, metadataB = %v!",
```
[the remark this answers is not in the document]
Comment (jackhaibo) [reply csi/server/plugin/opensds/controller.go]: Done.

File: csi/server/plugin/opensds/controller.go [hunk 5]
```diff
@@ -378,20 +362,20 @@ func isStringMapEqual(metadataA, metadataB map[string]string) bool {
 }
 
 // isVolumePublished Check if the volume is published and compatible
-func isVolumeCanBePublished(canAtMultiNode bool, attachReq *model.VolumeAttachmentSpec,
+func (c *Plugin) isVolumeCanBePublished(canAtMultiNode bool, attachReq *model.VolumeAttachmentSpec,
 	volMultiAttach bool) error {
 
-	glog.V(5).Infof("start to isVolumePublished, canAtMultiNode = %v, attachReq = %v",
+	glog.V(5).Infof("start to judge whether volume can be published, canAtMultiNode = %v, attachReq = %v",
```
[the remark this answers is not in the document]
Comment (jackhaibo) [reply csi/server/plugin/opensds/controller.go]: Done.

File: csi/server/plugin/opensds/controller.go [hunk 6]
```diff
@@ -421,56 +405,50 @@ func isVolumeCanBePublished(canAtMultiNode bool, attachReq *model.VolumeAttachme
...
 			req.VolumeId, err)
 		glog.Error(msg)
 		return nil, status.Error(codes.NotFound, msg)
 	}
 
-	pool, err := client.GetPool(volSpec.PoolId)
+	pool, err := c.Cli.GetPool(volSpec.PoolId)
 	if err != nil || pool == nil {
-		msg := fmt.Sprintf("the pool %s is not exist, error info: %v",
+		msg := fmt.Sprintf("the pool %s is not exist: %v",
```
[the remark this answers is not in the document]
Comment (jackhaibo) [reply csi/server/plugin/opensds/controller.go]: Done.

File: csi/server/plugin/opensds/controller.go (hunk 1 above)
Comment (jackhaibo) [reply csi/server/plugin/opensds/controller.go]: Done.

File: csi/server/plugin/opensds/controller.go (hunk 2 above)
Comment (jackhaibo) [reply csi/server/plugin/opensds/controller.go]: Done.

File: csi/server/plugin/opensds/controller.go (hunk 3 above)
Comment (jackhaibo) [reply csi/server/plugin/opensds/controller.go]: Done.

**Comment by wisererik** [comment]
@jackhaibo, @leonwanghui please squash commits

**Pull request merged**